omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -91,3 +91,10 @@
 </form>
 
 <div id="patient-search-results"></div>
+
+<% if (showRegisterPatientLink == "true") { %>
+<div>
+${ ui.message("coreapps.findPatient.registerPatient.label") }&nbsp;&nbsp;<a id="patient-search-register-patient" class="button" href="${ui.pageLink("registrationapp", "registerPatient", ["appId":"referenceapplication.registrationapp.registerPatient"])}">${ui.message("registrationapp.registration.label")}</a>
```

Comment:
@djazayeri There is no "main" register patient app - unless this is the one registrationapp.basicRegisterPatient 

Also see https://talk.openmrs.org/t/managing-app-definitions-and-extensions-in-openmrs/9601